protobuf/internal/proto_compile.bzl

```diff
@@ -577,17 +719,19 @@ def _proto_compile_impl(ctx):
       _build_grpc_invocation(run, builder)
       _build_grpc_out(run, builder)
 
+  _add_well_known_proto_imports(data, builder)
```

Comment:
It feels odd to add the well_known_protos for all proto_compile rules.  I don't know of any other protobuf tool that does that.  Is there a compelling reason to add it for everything?